jackmatchmaker/jacklib.py

```diff
@@ -1243,9 +1244,26 @@ def get_time():
 # -------------------------------------------------------------------------------------------------
 # Misc
 
+_error_callback = None
+
 jlib.jack_free.argtypes = [c_void_p]
 jlib.jack_free.restype = None
 
+try:
+    jlib.jack_set_error_function.argtypes = [JackErrorCallback]
+    jlib.jack_set_error_function.restype = None
+except AttributeError:
+    jlib.jack_set_error_function = None
+
+
+def set_error_function(error_callback):
+    global _error_callback
+    if jlib.jack_set_error_function:
+        _error_callback = JackErrorCallback(error_callback)
+        jlib.jack_set_error_function(_error_callback)
+
+    return -1
```

Comment:
This line is superfluous (sorry overlooked that inn my first review). Just let the function return `None` by default, i.e. remove the line.

Reply:
If you meant just remove `return -1`, then done.